README.md

```diff
@@ -49,18 +51,17 @@ Component | Spec | MDC Web
 
 > _NOTE:_ If you haven't used `create-react-app` before, you may want to read the [Overview Guide](https://github.com/facebook/create-react-app#quick-overview).
 
-
 #### Step 1: Install create-react-app
 
 Install `create-react-app`:
 
 ```
-npm i -g create-react-app
-create-react-app my-app
+npx create-react-app my-app
 cd my-app
+npm start
 ```
 
-> NOTE: all npm commands can be replaced with yarn
+> _NOTE:_ all npm commands can be replaced with yarn
```

Comment:
Does yarn have an equivalent to `npx`?

Reply:
not completely equivalent, but they do have `yarn create`

https://yarnpkg.com/lang/en/docs/cli/create/

Want to document this too?